### PR TITLE
[MDS-480] Bottom sheet v2

### PR DESCRIPTION
**Pull request by tkullisaar**

### CODE REVIEW

File: assets/js/hooks/dragger.js [hunk 1]
```diff
@@ -0,0 +1,118 @@
+
+ const addEvent = (() => {
+	if (document.addEventListener) {
```
Comment (margusp2heathmont) [inline assets/js/hooks/dragger.js:3]: Please use JS formatter with 2 or 4 space.

File: assets/js/hooks/dragger.js [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+                (this.el.dataset.is_open === undefined);
+                startedSwipe = false;
+            } else if (startedSwipe && touch.pageY > panelHeight / 2) {
+                console.log("open panel");
+                (this.el.dataset.is_open !== undefined);
+                startedSwipe = false;
+            }
+        });
+    },
+
```
Comment (phgrey) [inline assets/js/hooks/dragger.js:108]: too many empty lines

File: lib/moon/design/bottom_sheet.ex [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+      "md",
+      "lg"
+    ],
+    default: "md"
+  )
+
+  slot(panel, required: true)
+  slot(backdrop)
+
+  prop(testid, :string)
```
Comment (phgrey) [inline lib/moon/design/bottom_sheet.ex:22]: please add prop class and use it

**Review by phgrey — APPROVED** [review]
formattings, one more class attribute & merge it. please  do not forget to use `feat:...` instead of `fix:...` for commit message.

**Pull request merged**